tests/test_add_new_prebid_partner.py

```diff
@@ -185,6 +185,75 @@ def test_user_confirmation_accepted(self, mock_input,
     tasks.add_new_prebid_partner.main()
     mock_setup_partners.assert_called_once()
 
+  @patch('tasks.add_new_prebid_partner.setup_partner')
+  @patch('tasks.add_new_prebid_partner.input', return_value='y')
+  def test_prebid_medium_price_bucket_granularity(self, mock_input, mock_setup_partners, mock_dfp_client):
+    """
+    Sets prices buckets correctly based on prebid auto granularity
```

Comment:
Typo: should be prebid "medium" granularity

Reply:
Fixed